### PR TITLE
test on github action

### PR DESCRIPTION
**Pull request by markkong318**
# Change
- Use parallel running in bats (It uses GNU `parallel` command, need to install by brew in MacOS, but built-in in Github Action Environment)
- Parallel in 3 jobs, saving bat test from 15s->11s. Save max 4s. After some experiments, it is no different on 4~6 jobs 
- Update package.json

### CODE REVIEW

**Comment by markkong318** [comment]
by editing `cypress` in `bin` to run with bun
It has error on starting

image

**Comment by markkong318** [comment]
running bun and use time to measure the execution on local, it is no difference with node's testing speed

File: test/bats/lib/store.bash [hunk 1]
```diff
@@ -9,6 +9,30 @@ store_get() {
...
+    fi
+
+    sleep 1
+
+    VARIABLE="$(store_get $1)"
+  done
+
+  echo $VARIABLE
+}
+
```
Comment (markkong318) [inline test/bats/lib/store.bash:35]: there is one test need previous result, so implement the delay fetch to wait the result is shown (max 10 sec wait)

**Comment by markkong318** [comment]
For using bun to replace npm install, I found bun will change the `package-lock.json` file and some package inside  `@apollo/client` is not installed. So the test will get error

**Review by toruta39 — APPROVED** [review]
LGTM

**Pull request merged**